### ISSUE TITLE
Dataview filters for multifile property type don't work

### ISSUE THREAD

**Issue by justin8** [mdelobelle/metadatamenu#540]
When using a field of "multi" the data view query below will work and only list pages with the tag `#Person`. However when using the MultiFile (or "multiple links" depending on UI) field type the optional dataview query field does not appear to work using the same query. The documentation ([here](https://mdelobelle.github.io/metadatamenu/fields/#multi-file)) states:

> You can define : - a [dataviewjs query](https://mdelobelle.github.io/metadatamenu/fields/#dataview-query) returning an array of pages

Which made me think I shouldn't be using the `.map` and instead returning the raw objects, but that does not appear to work either. I can't seem to find what I'm doing wrong or find any examples of a working query with the multifile field type.

```
dv.pages("#Person").map(p => p.file.name)
```

**Comment by mdelobelle** [comment]
`dv.pages("#Person")` should be working just fine. Do you see any error in the console when accessing the field's modal?

**Comment by justin8** [comment]
No, no console errors appear. Even with a completely invalid query like `dsfasdfds` in the field settings; when clicking to show the modal and choose a link for that field I get no errors or any messages at all in the console.

**Comment by mdelobelle** [comment]
what if you try to paste the result of the query in the console?

can you write this dataviwjs block and paste here what is printed on the console?

~~~
```dataviewjs
console.log(dv.pages("#Person"))
```
~~~

**Comment by justin8** [comment]
It returns an array of 175 people from my notes.

![Screenshot 2024-02-20 at 10 52 59 am](https://github.com/mdelobelle/metadatamenu/assets/936320/e1471208-b4a5-4b3a-8819-bd3bc7e03567)

I didn't expand the notes for the obvious privacy reasons, but they're all notes tagged with #Person as expected.
